starter_code/server.js

```diff
@@ -17,14 +18,28 @@ app.get('/', (req, res) => {
     res.send("<h1>Hello from Pizza API</h1>");
 });
 
+app.get('/pizzas', async (req, res) => {
+    const pizzas = await Pizza.find(); 
+    res.json(pizzas) 
+})
+
 app.get('/customers', async (req, res) => {
-  const customers = [] // please fetch the customers from your database here, por favor!
-  res.json(customers) 
+    const customers = await Customer.find(); // please fetch the customers from your database here, por favor!
+    res.json(customers) 
 })
 
+app.get('/orders', async(req, res)=> {
+    const orders = await Order.find()
+    .populate({ path: 'customerInfo', select: 'address.city -_id'})
```

Comment:
Here you asked, how to make the ID of the order itself disappear.
It would be like this:
`Order.find().select('-_id').populate(...)`